hack/build/bazel-docker.sh

```diff
@@ -107,7 +107,7 @@ volumes="-v ${BUILDER_VOLUME}:/root:rw,z"
 mkdir -p "${HOME}/.docker"
 volumes="$volumes -v ${HOME}/.docker:/root/.docker:ro,z"
 
-if [[ $CDI_CRI = podman* ]] && [[ -f "${XDG_RUNTIME_DIR-}/containers/auth.json" ]]; then
+if [[ $CDI_CRI == podman* ]] && [[ -f "${XDG_RUNTIME_DIR-}/containers/auth.json" ]]; then
```

Comment:
So from what I understand, maybe do an `eq` instead would be better? I am afraid doing a single `=` is an assignment instead of a comparison.

Reply:
I did not touch this, it's the result of the sh formatter. I simply ran `make format` before making any changes.

Suposedly they're the same thing https://stackoverflow.com/questions/20449543/shell-equality-operators-eq

Reply:
I can still change it if you prefer so.